### ISSUE TITLE
[V2] Temporal AA doesn't work with the TextMesh

### ISSUE THREAD

**Issue by nxrighthere** [Unity-Technologies/PostProcessing#341]
##### What happened?
Temporal AA doesn't work with the [TextMesh](https://docs.unity3d.com/ScriptReference/TextMesh.html) component. This is especially noticeable in motion.

![TestText](https://s1.postimg.org/3rplaq7i8f/texttext.png)
##### Post-processing stack version (v1, v2)?
v2
##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
Unity 2017.1.1p4, Windows 10, PC Standalone (DX11, HDR, Linear, Deferred).

**Comment by Chman** [comment]
That's because text meshes are transparent and don't have any motion vectors. There's no good solution for this at the moment. Eventually we'll probably add an option to do TAA on opaque only.

**Comment by nxrighthere** [comment]
Yep, I just made my custom shader with an alpha cutoff for the TextMesh as a temporary solution for this issue.
